Makefile

```diff
@@ -121,11 +121,16 @@ docs/lint: | tfdocs-awk/install guard/program/terraform-docs
 	@ echo "[$@] documentation linting complete!"
 
 terratest/install: | guard/program/go
-	cd tests && go mod init {{cookiecutter.module_name}}/tests
```

Comment:
Not specific to this section, but this Makefile doesn't pin the tfdoc-awk repo

Reply:
fixed